microsoft-edge/webview2/release-notes.md

```diff
@@ -147,7 +147,149 @@ The following APIs have been promoted from Experimental to Stable in this Prerel
 ###### Runtime-only
 ###### SDK-only
 -->
-<!-- end of Jan 2024 Prerelease SDK -->
+<!-- end of Mar. 2024 Prerelease SDK -->
+
+
+<!-- ====================================================================== -->
+## 1.0.2415-prerelease
+
+Release Date: February 21, 2024<!-- todo: update date here & at top of file -->
+
+[NuGet package for WebView2 SDK 1.0.2415-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2415-prerelease)
+
+For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 122.0.2415.0 or higher.<!-- todo: fix version # -->

```

Comment:
"requires the WebView2 Runtime that ships with Microsoft Edge version 122.0.2415.0"
@victorhuangwq @peiche-jessica - pls check version # because 122 was already used last time, and there are some jumps:
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/cc933a6e-5f51-4892-ba1e-afe3687b4001)

Reply:
For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 123.0.2415.0 or higher.